reuse_test.go

```diff
@@ -0,0 +1,39 @@
+package testcontainers_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+func TestGenericContainer_stop_start_withReuse(t *testing.T) {
+	containerName := "my-postgres"
```

Comment:
nit: change to my-nginx to avoid confusion given the image used.

Reply:
Oh yes, I started with postgres, but reduced the test size. Changing it asap